src/dynamichmc_inference.jl

```diff
@@ -43,39 +43,35 @@ function dynamichmc_inference(prob::DiffEqBase.DEProblem, alg, likelihood, prior
                               ϵ=0.001, initial=Float64[], num_samples=1000,
                               kwargs...)
     P = DynamicHMCPosterior(alg, prob, likelihood, priors, kwargs)
+    println(typeof(transformations))
+    prob_transform(P::DynamicHMCPosterior) = as((transformations))
```

Comment:
No need for `as(...)` here, that's the api for *making* transformations. I would just make the user to that and work with the transformation object from then on.

Reply:
I felt this would give a cleaner interface, this can be changed later on too though so I'll keep this in mind for sure.

Reply:
What I am saying is that defining a function just for the sole purpose of returning the `transformations` makes little sense. I would do something like
```julia
function dynamichmc_inference(prob::DiffEqBase.DEProblem, alg, likelihood, priors, transformations;
                              ϵ=0.001, initial=Float64[], num_samples=1000,
                              kwargs...)
    P = DynamicHMCPosterior(alg, prob, likelihood, priors, kwargs)
    PT = TransformedLogDensity(transformations, P)
    PTG = FluxGradientLogDensity(PT);

    chain, NUTS_tuned = NUTS_init_tune_mcmc(PTG,num_samples, ϵ=ϵ)
    posterior = transform.(Ref(PTG.transformation), get_position.(chain));

    return posterior, chain, NUTS_tuned
end
```
instead.

Reply:
Oh I see.